x-pack/plugins/enterprise_search/public/applications/shared/kibana_breadcrumbs/set_breadcrumbs.test.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { SetAppSearchBreadcrumbs } from '../kibana_breadcrumbs';
+import { mountWithKibanaContext } from '../../test_utils/helpers';
+
+jest.mock('./generate_breadcrumbs', () => ({
+  appSearchBreadcrumbs: jest.fn(),
+}));
+import { appSearchBreadcrumbs } from './generate_breadcrumbs';
+
+jest.mock('react-router-dom', () => ({
+  useHistory: () => ({
+    createHref: jest.fn(),
+    push: jest.fn(),
+    location: {
+      pathname: '/current-path',
+    },
+  }),
+}));
+
+import { KibanaContext } from '../..';
```

Comment:
I think you can lose this line now that you're using `mountwithKibanaContext`, right?
```suggestion

```